python/graphstorm/config/argument.py

```diff
@@ -284,10 +284,16 @@ def verify_arguments(self, is_train):
             _ = self.use_node_embeddings
             _ = self.num_bases
             _ = self.num_heads
+            _ = self.num_ffn_layers_in_gnn
 
         _ = self.return_proba
         _ = self.alpha_l2norm
 
+
+        # ngnn
+        _ = self.num_ffn_layers_in_input
+        _ = self.num_fnn_layers_in_decoder
```

Comment:
`num_ffn_layers_in_decoder` instead of `num_fnn_layers_in_decoder`?

Reply:
updated